labels-config-service-impl/src/main/java/org/hypertrace/label/config/service/LabelsConfigServiceImpl.java

```diff
@@ -136,6 +139,59 @@ public void createLabel(
     }
   }
 
+  @Override
+  public void bulkCreateLabels(
+      BulkCreateLabelsRequest bulkCreateLabelsRequest,
+      StreamObserver<BulkCreateLabelsResponse> responseObserver) {
+    try {
+      RequestContext requestContext = RequestContext.CURRENT.get();
+      Lock labelsLock = this.stripedLabelsLock.get(requestContext.getTenantId());
```

Comment:
:( We've got a fairly high priority ticket to get rid of this locking - I suppose this is the same as the single create label lock and will be removed at the same time?

Reply:
Yes,it will be